lib/execution_engine2/db/models/models.py

```diff
@@ -320,7 +320,7 @@ class Job(Document):
     job_input = EmbeddedDocumentField(JobInput, required=True)
     job_output = DynamicField()
     condor_job_ads = DynamicField()
-    child_jobs = ListField()
+    child_jobs = ListField(null=True)
```

Comment:
Personally I prefer the empty list - that means you can just iterate over it without needing to do the None check.

Reply:
A non batch job shouldn't have child jobs though, right?

Reply:
It's not a huge deal though